frontend/src/app/tools/[toolName]/not-found.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import { useEffect } from "react";
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+
+export default function NotFound({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string };
+  reset: () => void;
+}) {
+  useEffect(() => {
+    // Log the error to an error reporting service
+    console.error(error);
+  }, [error]);
+
```

Comment:
So we need to keep this one in here? I would remove it everywhere.


https://github.com/user-attachments/assets/b451be3e-1329-4431-8650-76309ce47e86